### ISSUE TITLE
SVG images do not stay in frame in tabs

### ISSUE THREAD

**Issue by Nadine2016** [colinwilson/lotusdocs#32]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

When I add an SVG image within the tab shortcode, the image displays but is running out of the frame. PNGs seem to stay within the frame. 

### Expected behavior

Have SVG images behave the same as PNG images in a tab. 

### Screenshots

<img width="1345" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/17709352/21e71eaa-13c1-488a-94ae-e691a375b21b">

### Environment

```markdown
- OS: MacOS
- Browser: Chrome
- Browser Version: 116
```

**Comment by colinwilson** [comment]
Good catch. 👍

I'll push an update to fix this soon.

**Comment by Nadine2016** [comment]
Thank you so much!

**Comment by colinwilson** [comment]
@Nadine2016 Are you able to provide a sample SVG image file?

I'm having trouble replicating the behaviour in your screenshot with the few SVG images I've tested.

Also, are you inserting the SVG using regular markdown syntax? e.g. `![my image](/my/image/path/file.svg)`

**Comment by Nadine2016** [comment]
@colinwilson 

this is the code that I am using 

```
{{< tabs tabTotal="2">}}
{{% tab tabName="GCP" %}}
This is content for GCP. 

![Testimage](/img/core-concepts-apps.svg)
{{% /tab %}}
{{% tab tabName="EKS" %}}
This is content for EKS. 


| Value | Description | 
| -- | -- | 
| Value 1 | Description | 
| Value 2 | Description | 

{{% /tab %}}
{{< /tabs >}}
```

**Comment by Nadine2016** [comment]
![core-concepts-apps](https://github.com/colinwilson/lotusdocs/assets/17709352/32d1f681-3cc2-47d9-9b4d-74f1a297569d)
 
And this is an image.

**Comment by Nadine2016** [comment]
<img width="1404" alt="image" src="https://github.com/colinwilson/lotusdocs/assets/17709352/a978b739-85f5-4cb5-ab0b-7123fd27f3df">

And this is how it looks like for me

**Comment by colinwilson** [comment]
@Nadine2016 Thanks for additional info 👍

Fixed by 5e4c8f1
